modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/util/session.es.js

```diff
@@ -28,6 +28,12 @@ function getSessionClickFormData(cmd) {
 		formData.append('doAsUserId', doAsUserIdEncoded);
 	}
 
+	if (options && typeof options === 'object') {
+		Object.entries(options).forEach(([key, value]) => {
```

Comment:
I'm not sure we want to unconditionally do this. This creates an API with the same problem we had in the original one where these 2 calls have very similar effects:

```javascript
Liferay.Util.Session.set(
    'foo', 
    'bar', 
    {
        useHttpSession: true
    }
);

Liferay.Util.Session.set(
    '',
    '',
    {
        foo: 'bar',
        useHttpSession: true
    }
);
```